woofbot/umbrel-app.yml

```
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: woofbot
+category: Explorers
+name: WoofBot
+version: "0.1.0"
+tagline: A chatbot for your personal node.
+description: >-
+  WoofBot is a chat bot that runs on your personal Bitcoin node and sends telegram notifications
+  based on pre-configured conditions: addresses, transactions, new-blocks and price changes.
+developer: WoofBotApp
+website: https://github.com/woofbotapp
+dependencies:
+  - bitcoin
+repo: https://github.com/woofbotapp/woofbotapp
+support: https://github.com/woofbotapp/woofbotapp/discussions
+port: 8092
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
```

```suggestion
deterministicPassword: true
```

I see that you use a deterministic password for Woofbot using `$APP_PASSWORD`. In order for this password to be displayed to the user (in the app store) you should use this flag instead.

---

Hi @nevets963 , in version 0.2.0 I've added an automatic login when $APP_PASSWORD is undefined (the frontend automatically tries to login with an empty password). If the user chooses to change the password manually, then obviously the password will be required.

So for the initial login we don't need Umbrel to set APP_PASSWORD to a deterministic-password, and we can allow it to be empty, because Umbrel will protect the app using the user's Umbrel password - right?

---

Yes exactly, in Umbrel v0.5 (and onwards) we protect all apps automatically with the user's password and 2FA (where applicable).

I just tested your `docker-compose.yml` file dropping the line `$APP_PASSWORD` and it works :) You can ignore my suggestion. Can you update your PR to drop that line, then I can approve :)

---

Thanks! I removed the line.